io.gitlab.Turtlico.json

```diff
@@ -10,10 +10,14 @@
         "--socket=x11",
         "--socket=wayland",
         "--socket=pulseaudio",
+        "--filesystem=xdg-run/dconf",
```

Comment:
Do you really need the explicit dconf access? Since you didn't before and it's generally not necessary in the current runtime

https://docs.flatpak.org/en/latest/sandbox-permissions.html?highlight=DCONF#dconf-access

Reply:
No, it should not be needed. It's just another thing from an old template that was kept in the manifest in the upstream repository. Sorry about that.